app/subscriber/src/features/settings/MyMinisterSettings.tsx

```diff
@@ -15,7 +15,7 @@ export const MyMinisterSettings: React.FC = () => {
 
   let activeMinisters = ministers
     .filter((m) => m.isEnabled)
-    .sort((a, b) => a.name.localeCompare(b.name));
+    .sort((a, b) => a.sortOrder - b.sortOrder);
```

Comment:
Wouldn't this only sort by the `sortOrder`?  Doesn't the other function also sort by the `name`?